tests/test_utils.py

```diff
@@ -298,11 +306,536 @@ def check_multistream_classic(fnames):
+
+    return snotel_swe
+
+
+def read_vic_42_output(lat, lng, science_test_data_dir, items):
+    ''' Reads output from VIC 4.2.
+    '''
+
+    if items['compare_to'] == 'ecflux':
+        vic_42_file = 'en_bal_%s_%s' % (lat, lng)
+        vic_42_dir = os.path.join(science_test_data_dir, 'test_runs',
+                                  items['archive'], 'ecflux', 'results')
+
+    elif items['compare_to'] == 'snotel':
+        vic_42_file = 'outfile_%s_%s' % (lat, lng)
+        vic_42_dir = os.path.join(science_test_data_dir, 'test_runs',
+                                  items['archive'], 'snotel', 'results')
+
+    else:
+        raise ValueError("this option (%s) has not yet been implemented"
+                         % items['compare_to'])
+
+    vic_42 = pd.read_csv(os.path.join(vic_42_dir, vic_42_file),
+                         sep='\t',
+                         skiprows=5)
+
+    # remove comment sign from column names in DataFrame
+    vic_42 = vic_42.rename(columns=lambda x: x.replace('#', ''))
+
+    # remove spaces from column names in DataFrame
+    vic_42 = vic_42.rename(columns=lambda x: x.replace(' ', ''))
+
+    # rename radiation variables to be consistent with VIC 5
+    if items['compare_to'] == 'ecflux':
+        vic_42 = vic_42.rename(columns=lambda x: x.replace('OUT_NET_SHORT',
+                               'OUT_SWNET'))
+        vic_42 = vic_42.rename(columns=lambda x: x.replace('OUT_NET_LONG',
+                               'OUT_LWNET'))
+
+    # add datetime index
+    time_cols = ['YEAR', 'MONTH', 'DAY']
+    vic_42.index = pd.to_datetime(vic_42[time_cols])
+
+    if 'HOUR' in vic_42:
+        vic_42['SEC'] = vic_42.HOUR * 3600
+        vic_42.index += pd.Series([pd.Timedelta(s, unit='s') for s in
+                                  vic_42['SEC']],
+                                  index=vic_42.index)
+        time_cols.extend(['SEC', 'HOUR'])
+
+    # remove year, day columns of DataFrame
+    vic_42.drop(time_cols, inplace=True, axis=1)
+
+    return vic_42
+
+
+def read_vic_5_output(lat, lng, science_test_data_dir, items):
+    ''' Read VIC 5.0.x output.
+    '''
+
+    if items['compare_to'] == 'ecflux':
+        vic_5_file = 'en_bal_%s_%s.txt' % (lat, lng)
+        vic_5_dir = os.path.join(science_test_data_dir, 'test_runs',
+                                 items['archive'], 'ecflux', 'results')
+
+    elif items['compare_to'] == 'snotel':
+        vic_5_file = 'outfile_%s_%s.txt' % (lat, lng)
+        vic_5_dir = os.path.join(science_test_data_dir, 'test_runs',
+                                 items['archive'], 'snotel', 'results')
+
+    else:
+        raise ValueError("this option (%s) has not yet been implemented"
+                         % items['compare_to'])
+
+    vic_5 = pd.read_csv(os.path.join(vic_5_dir, vic_5_file),
+                        skiprows=2,
+                        sep='\t')
+
+    # remove spaces from column names
+    vic_5 = vic_5.rename(columns=lambda x: x.replace(' ', ''))
```

Comment:
since you are overwritting the orginal `vic_5`, better to just use the `inplace` keyword argument:

```Python
vic_5.rename(columns=lambda x: x.replace(' ', ''), inplace=True)
```